camayoc/tests/qcs/api/v1/utils.py

```diff
@@ -56,10 +56,28 @@ def wait_until_state(scan, timeout=120, state='completed'):
                 .format(
                     scan._id,
                     state,
+                    pprint.pformat(scan.read().json()),
+                    pprint.pformat(scan.results().json()),
                 )
```

Comment:
This fixes that problem jenkins was running into.

After fixing this, the satellite scans are still timeing out but that is because of https://github.com/quipucords/quipucords/issues/569 ! So they should fail.